connectors/kibana.py

```diff
@@ -151,6 +151,7 @@ async def prepare(service_type, index_name, config, connector_definition=None):
             "last_sync_status": None,
             "last_sync_error": None,
             "last_sync_scheduled_at": None,
+            "last_permissions_sync_scheduled_at": None,
```

Comment:
This can also be added to 
1. https://github.com/elastic/connectors-python/blob/main/tests/fixtures/connector.json
2. https://github.com/elastic/connectors-python/blob/main/tests/sources/fixtures/mysql/connector.json
3. https://github.com/elastic/connectors-python/blob/main/tests/sources/fixtures/mongodb/connector.json

But not a big issue as they will be evaluated to `None` without adding.